packages/utils/package.json

```diff
@@ -37,7 +37,6 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@chainsafe/as-sha256": "^1.1.0",
-    "any-signal": "3.0.1",
```

Comment:
instead of replacing any-signal with a custom implementation, we could consider upgrading it to latest version [v4.1.1](https://github.com/jacobheun/any-signal/releases/tag/v4.1.1) but need to make sure it doesn't break our code as it's a new major version